### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by vvoo21**
Hi 🙋‍♀️,
In this pull request, I used a relational database to create the initial data structure for a vet clinic.

Used the [template](https://github.com/microverseinc/curriculum-template-databases) to generate the repository.
Created a database named vet_clinic.
Created a table named animals with the columns (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
Inserted the given data into the table
Wrote queries for the given quests.
Copied the SQL used to create the table into the schema.sql file in the generated repository.
Copied the SQL used to insert data to the table into the data.sql file in the generated repository.
Copied the SQL used to perform queries into the queries.sql file in the generated repository.

####  Screenshot of the results of my queries:

queries 1-2

queries 3-5

queries 6-8

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @vvoo21, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file.

- [ ] Please fix the broken image in the `Database Schema Diagram` section of your README file.
- [ ] Please remove the Microverse logo from your README file.
- [ ] This project is a solo project. Please remove the second `Author` from you README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by vvoo21** [comment]
Thanks @CollinsTatang !

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+  id INT GENERATED BY DEFAULT AS IDENTITY,
```
Comment (Alfredbis29) [inline schema.sql:4]: - [ OPTIONAL] I would highly recommend using the PRIMARY KEY (id) in your database design. :wink: The primary key is a crucial element in a database table as it uniquely identifies each record in the table. By defining the id column as the primary key, you ensure that each row in the table has a unique identifier. This helps in maintaining data integrity and ensures that there are no duplicate records.

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi @vvoo21  
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * Descriptive Readme :white_check_mark: 
 * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] Check Under the review  

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (vvoo21) [reply schema.sql:4]: Thank you @Alfredbis29.

**Pull request merged**